stdlib/public/TensorFlow/OpaqueHandles.swift

```diff
@@ -25,11 +27,56 @@ public final class ResourceHandle {
 }
 
 /// `VariantHandle` is the type used by ops and the `#tfop()` syntax to
-/// represent TensorFlow "variant" values.  It exists only to represent edges
-/// in TensorFlow graphs, and has no host-side representation (and thus no
-/// methods).
+/// represent TensorFlow "variant" values.
+/// TODO: rename this source file as this is no longer "opaque".
 public final class VariantHandle {
-  private init() {
-    fatalError("VariantHandle is a marker type that can never be created")
+  public let cTensorHandle: CTensorHandle
+
+  @usableFromInline
+  init(owning cTensorHandle: CTensorHandle) {
+    self.cTensorHandle = cTensorHandle
+  }
+
+  deinit {
+    debugLog("De-initializing TensorHandle.")
+    TFE_DeleteTensorHandle(cTensorHandle)
+    debugLog("Returning from deinit of VariantHandle.")
+  }
+}
+
+extension VariantHandle : TensorSendableReceivable {
+  @inlinable
+  static func receiveFromAccelerator(_ computation: _TensorComputation,
+                                     _ tensorID: Int
+  ) -> VariantHandle {
```

Comment:
Break lines like this:
```swift
  static func receiveFromAccelerator(
    _ computation: _TensorComputation,
    _ tensorID: Int
  ) -> VariantHandle {
```

https://google.github.io/swift/#function-declarations

Reply:
Done